endpointmanager/README.md

```diff
@@ -202,5 +204,28 @@ The Endpoint Manager expects the format of an endpoint source list to be in the
 }
 ```
 
-Exceptions:
-* Cerner
\ No newline at end of file
+Cerner:
+
+```
+{
+  "endpoints": [
+    {
+      "name": <name of the organization>,
+      "baseUrl": <location of the FHIR endpoint>,
+      "type": <endpoint type>
+    },
+    ...
+  ]
+}
+```
+
+NPPES:
+
+```
+"NPI","Endpoint Type","Endpoint Type Description","Endpoint","Affiliation","Endpoint Description","Affiliation Legal Business Name","Use Code","Use Description","Other Use Description","Content Type","Content Description","Other Content Description","Affiliation Address Line One","Affiliation Address Line Two","Affiliation Address City","Affiliation Address State","Affiliation Address Country","Affiliation Address Postal Code"
+...
+```
+
+### Adding a New Endpoint List
+
+To add a new endpoint list, add an entry to the EndpointResourcesList.json file located in the resources/prod_resources directory with the endpoint name, the name the endpoint source file will be saved as, and the endpoint URL. If the format does not match any of those listed above in the expected endpoint formats, add a new parser.
```

Comment:
Add more info about adding a new parser. eg: `See lantern-back-end/endpointmanager/pkg/fetcher/cernerlist.go or  lantern-back-end/endpointmanager/pkg/fetcher/epiclist.go for examples of the interface which endpoint list parsers need to adhere to.`